cmd/cofidectl/cmd/federation/federation.go

```diff
@@ -18,6 +18,14 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	FederationStatus_HEALTHY   string = "Healthy"
```

Comment:
nit: Unsure about all upper case in go variable names, I'd probably have gone for FederationStatusHealthy

Reply:
This was partly inspired by my recent time in the Istio API, which has consts like `ServerTLSSettings_PASSTHROUGH`, `ServerTLSSettings_MUTUAL`

https://pkg.go.dev/istio.io/api/networking/v1#ServerTLSSettings_TLSmode

Although looking now at these API defs, these are mostly generated definitions so perhaps not good to emulate this manually. I'll change the new consts over to Pascal case

Reply:
Amended